mtools/mlaunch/mlaunch.py

```diff
@@ -328,10 +328,7 @@ def run(self, arguments=None):
                                        'required to run the stop command '
                                        '(requires --auth, default="%s")'
                                        % ' '.join(self._default_auth_roles)))
-        init_parser.add_argument('--auth-role-docs', action='store_true',
-                                 default=False,
-                                 help='auth-roles are json documents')
-        init_parser.add_argument('--no-initial-user', action='store_false',
+        init_parser.add_argument('--initial-user', action='store_false',
```

Comment:
If `auth-role-docs` option is removed, we can also remove the argument parser logic as well (around line 750)

Reply:
I re-added `auth-role-docs` instead.